modules/nodes-scheduler-taints-tolerations-adding-machineset.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-scheduler-taints-tolerations.adoc
+// * post_installation_configuration/node-tasks.adoc
+
+[id="nodes-scheduler-taints-tolerations-adding-machineset_{context}"]
+= Adding taints and tolerations using a MachineSet
+
+You can add taints to nodes using a Machineset. All nodes associated with the `MachineSet` are updated with the taint. Pods and tolerations respond the same as a taint added directly to the nodes.
+
+.Procedure
+
+. Add a toleration to a pod by editing the `Pod` specification to include a `tolerations` section:
++
+.Sample pod configuration file with `Equal` operator
+[source,yaml]
+----
+spec
+...
+  tolerations:
+  - key: "key1" <1>
+    operator: "Equal"
+    value: "value1"
+    effect: "NoExecute"
+    tolerationSeconds: 3600 <2>
+----
+<1> The toleration parameters, as described in the taint and toleration components table.
+<2> The `tolerationSeconds` parameter specifies how long a pod can remain bound to a node before being evicted.
++
+For example:
++
+.Sample pod configuration file with `Exists` operator
+[source,yaml]
+----
+spec
+...
+  tolerations:
+  - key: "key1"
+    operator: "Exists"
+    effect: "NoExecute"
+    tolerationSeconds: 3600
+----
+
+. Add the taint to the `MachineSet`:
+
+.. Edit the `MachineSet` YAML for the nodes you want to taint or create a new `MachineSet`:
```

Comment:
I don't think you can create a new machine set with `oc edit`, this is for only existing objects. Is this a conditional step? Or perhaps you can punt and simply omit _or create..._ entirely.